src/openloco/graphics/gfx.cpp

```diff
@@ -16,7 +17,59 @@ namespace openloco::gfx
     // 0x0044733C
     void load_g1()
     {
-        call(0x0044733C);
+        auto g1Path = environment::get_path(environment::path_id::g1);
+
+        FILE * file;
+        gfx::loco_g1_header header;
+
+        loco_g1_element * g1Elements;
+        void * g1Buffer;
+
+        file = fopen(g1Path.make_preferred().u8string().c_str(), "rb");
+
+        if (file != NULL) {
+            if (fread(&header, 8, 1, file) == 1) {
+
+                // Read element headers
+                g1Elements = (loco_g1_element *)malloc(header.num_entries * sizeof(loco_g1_header));
```

Comment:
You won't be able to do this until all references to g1Elements are implemented or replaced.
You will need to set it to `0x9E2424`

Reply:
Duh. Guessing we should have a define for that, akin to `RCT2_ADDRESS_G1_ELEMENTS`?

Reply:
Yes, `loco_global_array<loco_g1_element, 15000` etc.